bluesky_widgets/models/auto_plot_builders/_base.py

```diff
@@ -73,6 +73,14 @@ def handle_figure_removed(self, figure):
                 if figure is plot_builder.figure:
                     self.plot_builders.remove(plot_builder)
             elif hasattr(plot_builder, "figures"):
+                # This code looks broken, was untested, and TAC is not
```

Comment:
I can't figure out what looks broken about this. I think it works and is being used.